src/_includes/layouts/page.njk

```diff
@@ -20,9 +20,6 @@
     <a href="/views/" class="blue card">
       <h2>Read current views in inclusive data science</h2>
     </a>
-    <a href="/learn/" class="green card">
-      <h2>Search the We Count Library</h2>
-    </a>
```

Comment:
The card "Search the We Count Library" is removed from the home page. Is there a request for it?

Reply:
Good catch! As we discussed in a call, this was removed in error while getting rid of all references to the previous "learn" page name. I'll restore it :)